jvcl/run/JvFullColorCtrls.pas

```diff
@@ -742,6 +742,9 @@ implementation
 uses
   RTLConsts, TypInfo, Forms,
   JclMath, JclLogic, // For EnsureRange and Min/Max
+  {$IFDEF HAS_UNIT_SYSTEM_UITYPES}
+  UITypes,
```

Comment:
The UnitName should really be prefixed, the compiler supports it.
So please replace `UITypes` by `System.UITypes`